bluemira/codes/interface.py

```diff
@@ -139,6 +141,51 @@ def set_parameters(self, params):
         self.__recv_mapping = get_recv_mapping(params, NAME)
         self.__send_mapping = get_send_mapping(params, NAME)
 
+    def _get_new_inputs(self, remapper: Optional[Union[callable, Dict]] = None):
```

Comment:
I think this should be the type `Callable` rather than the function. It'll need importing from `typing`
```suggestion
    def _get_new_inputs(self, remapper: Optional[Union[Callable, Dict]] = None):
```

Reply:
Done